core/src/main/java/nl/nn/adapterframework/pipes/UnzipPipe.java

```diff
@@ -337,14 +337,12 @@ public void setCreateSubdirectories(boolean b) {
 	public boolean isCreateSubdirectories() {
 		return createSubdirectories;
 	}
-	
-	@IbisDoc({"if set <code>true</code>, validation on directory is ignored", "false"})
-	public boolean isCheckDirectory()
-	{
-		return checkDirectory;
+
+	@IbisDoc({"if set <code>true</code>, validation of directory is ignored", "false"})
+	public boolean isAssumeDirectoryExists() {
+		return assumeDirectoryExists;
```

Comment:
Could you:
- set the setter before the getter, with the ibisdoc before the setter
- make a setCheckDirectory() with \@Deprecated, and a config warning stating what you did